main.go

```diff
@@ -804,7 +804,7 @@ func main() {
 		fmt.Printf(serviceInfo())
 		addr := fmt.Sprintf(":%d", *port)
 		http.HandleFunc("/", welcome)
-		http.ListenAndServe(addr, http.HandlerFunc(router))
+		log.Fatal(http.ListenAndServe(addr, http.HandlerFunc(router)))
```

Comment:
Or perhaps, better yet, you should use the explicit `os.Stderr` as target and call `fmt.Fprintf` function, followed by an `os.Exit(1)` call as it is done in some other places.